### PR TITLE
R20.1 286

### PR DESCRIPTION
**Pull request by dougkenik**
Fix for #286 and the issue Jaime reported when opening a saved project and Cura silently erroring out

### CODE REVIEW

**Review by jake-kirkland-teton — APPROVED** [review]
Seems like it is functioning as expected. One question about functionality that may or may not be an issue. So when a model is copied, the anchors/loads also copy, the loads just do not have an arrow anymore. But the modmesh that is created does in fact have anchors and loads of it's own. Is this enough of a problem to look into?

DuplicateModelGif

**Comment by dougkenik** [comment]
> Seems like it is functioning as expected. One question about functionality that may or may not be an issue. So when a model is copied, the anchors/loads also copy, the loads just do not have an arrow anymore. But the modmesh that is created does in fact have anchors and loads of it's own. Is this enough of a problem to look into?
> 
> DuplicateModelGif

Yeah, we should just remove those from the modifier mesh

**Comment by dougkenik** [comment]
Ok - we're good now

**Review by bradymadams — APPROVED** [review]

File: SmartSlicePlugin/stage/SmartSliceStage.py [hunk 1]
```diff
@@ -179,6 +179,15 @@ def onStageSelected(self):
...
             mesh.setSelectable(False)
 
+            # Remove any HighlightFace if they exist
+            for node in mesh.getChildren():
+                if isinstance(node, SmartSliceScene.HighlightFace):
+                    mesh.removeChild(node)
+                elif isinstance(node, SmartSliceScene.Root):
+                    mesh.removeChild(node)
+
+            a = 1
```
Comment (bradymadams) [inline SmartSlicePlugin/stage/SmartSliceStage.py:189]: Wait, what's `a`?

**Review by jake-kirkland-teton — APPROVED** [review]

**Pull request merged**